test/parallel/test-postmortem-metadata.js

```diff
@@ -9,6 +9,24 @@ const assert = require('assert');
 const { spawnSync } = require('child_process');
 const args = [process.execPath];
 
+// For shared lib case, check shared lib instead
+if (process.config.variables.node_shared) {
+  const path = require('path');
+  args.pop();
+  // There is no nm in Windows, maybe use dumpbin instead.
+  // However, we still try to compose the correct shared lib path
+  if (common.isWindows) {
+    args.push(path.join(path.dirname(process.execPath), 'node.dll'));
```

Comment:
I wonder if the logic for getting the shared lib instead of the stub executable is worth putting in the `test/common/shared-lib-util.js` being added in https://github.com/nodejs/node/pull/18626. The platform specific logic could then be kept out of these changes.

Reply:
sure. that would be great! Since that PR is not landed yet. I will do that after the merge.

Reply:
@richardlau per your suggestion and the PR is landed, I moved the shared lib path logic into `test/common/shared-lib-util.js`